File: UnrelatedAssessment/Addition.java
import java.util.*;

class Addition {
    public static void main(String[] args) {
        int a, b, c, d;

        try (Scanner s = new Scanner(System.in)) {
            System.out.println("Enter Value A: ");
            a = s.nextInt();
            System.out.println("Enter Value B: ");
            b = s.nextInt();
            System.out.println("Enter Value C: ");
            c = s.nextInt();
            System.out.println("Enter Value D: ");
            d =s.nextInt();
        }

        int num1 = (a + b);
        int num2 = (c + d);

        if (num1 > num2) {
            System.out.println(num1 + " is larger");
        } else {
            System.out.println(num2 + " is larger");
        }

    }
     
}

File: UnrelatedAssessment/Addition.cpp
#include <iostream>

using namespace std;

int main() {
    // Getting Inputs
    int a, b, c, d;

    cout << "Enter Value A: ";
    cin >> a;

    cout << "Enter Value B: ";
    cin >> b;

    cout << "Enter Value C: ";
    cin >> c;

    cout << "Enter Value D: ";
    cin >> d;

    // Calculating Values
    int num1, num2;

    num1 = (a + b);
    num2 = (c + d);

    // Statements
    if (num1 > num2) {
        cout << num1 << " is the largest number";
    } else {
        cout << num2 << " is the largest number";
    }

}